SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -31,5 +31,13 @@ public async Task<ActionResult<ArticleResponseDTO>> GetArticleByTitleAsync(strin
             var articleResponse = _mapper.Map<ArticleResponseDTO>(article);
             return Ok(articleResponse);
         }
+
+        public async Task<ActionResult> DeleteArticleAsync(string title)
+        {
+            bool deletedSuccessful = await _articleService.DeleteArticleAsync(title);
+            if (deletedSuccessful) return Ok("Article deleted successful");
```

Comment:
format please

Reply:
I wasnt sure how it would be best to leave it without the braces . So i will remove the empty line then

Reply:
before If - better to have an empty line.
return - should be in a new line, and actually for myself better to keep braces after if

Reply:
Okey